e2e-tests/portal/cypress/support/portal/utilisation-reports/assertUploadReportValidationErrorTableRowContains.js

```diff
@@ -0,0 +1,34 @@
+const { utilisationReportUpload } = require('../../../e2e/pages');
+
+module.exports = ({ tableRowIndex, message, exporter, row, column, entry }) => {
+  utilisationReportUpload
+    .validationErrorMessage(tableRowIndex)
+    .invoke('text')
+    .then((text) => {
+      expect(text.trim()).to.equal(message);
+    });
+  utilisationReportUpload
+    .validationErrorExporter(tableRowIndex)
+    .invoke('text')
+    .then((text) => {
+      expect(text.trim()).to.equal(exporter);
+    });
+  utilisationReportUpload
+    .validationErrorRow(tableRowIndex)
+    .invoke('text')
+    .then((text) => {
+      expect(text.trim()).to.equal(row);
+    });
+  utilisationReportUpload
+    .validationErrorColumn(tableRowIndex)
+    .invoke('text')
+    .then((text) => {
+      expect(text.trim()).to.equal(column);
+    });
+  utilisationReportUpload
+    .validationErrorValue(tableRowIndex)
+    .invoke('text')
+    .then((text) => {
+      expect(text.trim()).to.equal(entry);
+    });
```

Comment:
Can we use `cy.assertText` here please (new command recently merged) so each assertion here can just be e.g:

```js
cy.assertText(utilisationReportUpload.validationErrorValue(tableRowIndex), entry);
```

Reply:
I had a look for something like this and couldn't find it, have just merged in main and searched for `assertText` and it isn't finding any results... 🤔 Do you know where that is defined?

Reply:
Sorry @RLavender98 I thought the `assertText` PR had been merged - it's been merged now, if you pull the latest 🙏